benchmark/v5.0-performance-benchmarking-with-tpch.md

```diff
@@ -0,0 +1,123 @@
+---
+title: TiDB TPC-H 100GB Performance Test Report -- v5.0 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
+summary: Compare the TPC-H 100GB performance of TiDB 5.0 MPP mode, Greenplum 6.15.0, and Apache Spark 3.1.1.
+---
+
+# TiDB TPC-H 100GB Performance Test Report -- TiDB v5.0 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
+
+## Test overview
+
+This test aims at comparing the TPC-H 100GB performance of TiDB v5.0 in the MPP mode with that of Greenplum and Apache Spark, two mainstream analytics engines, in their latest versions. The test result shows that the performance of TiDB v5.0 in the MPP mode is two to three times better than that of the other two solutions.
```

Comment:
"two to three times better" is not a proper description. Should be 
"2 to 3 times faster than that of the other two solutions under TPC-H workload"